.github/scripts/generate-release-contributors.sh

```diff
@@ -87,4 +87,5 @@ echo $contributors1 $contributors2 \
   | grep -v dependabot \
   | grep -v codecov \
   | grep -v opentelemetrybot \
+  | grep -v opentelemetry-java-bot \
```

Comment:
I don't think this is used anymore?

Reply:
The contributors script picked it up, and [this commit](https://github.com/open-telemetry/opentelemetry-java/commit/90f84f6017175e48d5a67e7ccd934c3fda3551c9) used it 🤔